core/node/proof_data_handler/src/lib.rs

```diff
@@ -121,6 +126,15 @@ fn create_proof_processing_router(
                         .await
                 },
             ),
+        )
+        .route("/tee/get_proof/:l1_batch_number",
```

Comment:
I don't think it belongs here. We don't want to expose TEE prover API.
Instead, you need to make an RPC method in the our "main" API (some pointers -- `core/lib/web3_decl` and `core/node/api_server`).

Reply:
@popzxc Is adding my stuff to [`core/node/api_server/src/web3/backend_jsonrpsee`](https://github.com/matter-labs/zksync-era/tree/main/core/node/api_server/src/web3/backend_jsonrpsee) a good idea, or should it belong somewhere else?  Sorry for the basic question – I'm trying to understand which [layer](https://github.com/matter-labs/zksync-era/blob/2fa2249dca15b1968fceec11e485850395f03c9d/core/bin/zksync_server/src/node_builder.rs#L627-L707) I should be looking into ([`Web3ServerLayer`](https://github.com/matter-labs/zksync-era/blob/2fa2249dca15b1968fceec11e485850395f03c9d/core/node/node_framework/src/implementations/layers/web3_api/server.rs#L72-L94)?) and to grasp the overall structure of the RPC server.